client/my-sites/site-settings/form-writing.jsx

```diff
@@ -123,6 +124,23 @@ class SiteSettingsFormWriting extends Component {
 						<PressThis />
 					</div>
 				) }
+				{
```

Comment:
Let's move this card below the Composing card - it makes sense for it to be there, and that's how it's shown in the mockups in p6TEKc-HQ-p2.